README-offloaded-indexing.md

```diff
@@ -0,0 +1,79 @@
+### Cluster with remote indexing node, this a little manual since deploy vars are getting messy
+    $ export GIT_REL='v103rc2' && export ENCD_REL='v103rc2' && echo "$GIT_REL $ENCD_REL"
+
+    # Normal elasticsearch cluster
+    $ bin/deploy -b "$GIT_REL" --cluster-name "$ENCD_REL" --es-wait --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Head frontend node: 12 app workers, 8 index workers
+    $ export ES_IP=172.31.25.24
+    $ bin/deploy -b $GIT_REL --es-ip $ES_IP --cluster-name "$ENCD_REL" --remote-indexing --primary-indexing --index-procs 8 --app-workers 12 --pg-open --do-batchupgrade no --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Check environment vars
+    $ cat /etc/environment
+        * ENCD_APP_WORKERS=12
+        * ENCD_INDEX_PROCS=8
+        * ENCD_REMOTE_INDEXING=true
+
+    # ssh on and stop apache after the instance restart
+    $ sudo service apache2 stop
```

Comment:
This is unfortunate but by design the head node needs to be built first.  And will start indexing if the offloaded indexing node doesn't not exist.